src/libraries/Microsoft.Extensions.Caching.Abstractions/ref/Microsoft.Extensions.Caching.Abstractions.cs

```diff
@@ -103,6 +103,7 @@ public partial interface IMemoryCache : System.IDisposable
         Microsoft.Extensions.Caching.Memory.ICacheEntry CreateEntry(object key);
         void Remove(object key);
         bool TryGetValue(object key, out object value);
+        void Clear();
```

Comment:
@MCCshreyas - thanks for the contribution!

However, since this is public API we need to follow the API review process before accepting changes in this area. See the link at the top of this file: https://aka.ms/api-review process. We won't be able to merge this PR without first getting an API approval.

An issue with the currently proposed changes is that IMemoryCache is an interface, and adding more members to an interface is a binary breaking change. This type of change isn't acceptable in these libraries.

Reply:
So need to come up with new interface for a single method? 🤔